packages/venia-concept/src/components/ProductImageCarousel/thumbnail.js

```diff
@@ -10,19 +11,35 @@ class Thumbnail extends Component {
     static propTypes = {
         classes: PropTypes.shape({
             root: PropTypes.string
-        })
+        }),
+        item: imageItemPropType,
+        activeItemSrc: PropTypes.string,
+        onClickHandler: PropTypes.func,
+        handlerParams: PropTypes.string
+    };
+
+    onClickHandlerWrapper = () => {
+        const { onClickHandler, handlerParams } = this.props;
```

Comment:
Why do you need property handlerParams here if you already pass item as a property to the component?